### PR TITLE
[14.0][ADD] pos_receipt_hide_price

### PR DESCRIPTION
**Pull request by paradoxxxzero**
Adds a button in the receipt view to hide price from the receipt for printing and emailing purpose

### CODE REVIEW

File: pos_receipt_hide_price/static/src/js/OrderReceipt.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+odoo.define("pos_receipt_hide_price.OrderReceipt", function (require) {
+    "use strict";
+
+    const OrderReceipt = require("point_of_sale.OrderReceipt");
+    const Registries = require("point_of_sale.Registries");
+
+    const HidePriceOrderReceipt = (OrderReceipt) =>
+        class extends OrderReceipt {
+            constructor(_, {hpState}) {
```
Comment (legalsylvain) [inline pos_receipt_hide_price/static/src/js/OrderReceipt.js]: If I understand correctly hp is for hidePrice ? I think ``hidePriceState`` could be more explicit. don't you think ?

**Review by legalsylvain — COMMENTED** [review]
Hi @paradoxxxzero. 

This PR (and module) is depecrating https://github.com/OCA/pos/pull/682 don't you think ?

Some remarks inline.

Nice to have : Could you add a print screen in the description ? 

Unfortunately not owl expert : no JS review.

CC : @dsolanki-initos, @ivantodorovich

File: pos_receipt_hide_price/static/src/js/OrderReceipt.js (hunk 1 above)
Comment (paradoxxxzero) [reply pos_receipt_hide_price/static/src/js/OrderReceipt.js]: Agreed

**Comment by paradoxxxzero** [comment]
Hello @legalsylvain, thanks for the review.

I think it's a slightly different take solving the same problem so yes it may deprecate #682.

I'll update the description with some screenshots.

**Comment by ivantodorovich** [comment]
> Hello @legalsylvain, thanks for the review.
> 
> I think it's a slightly different take solving the same problem so yes it may deprecate #682.
> 
> I'll update the description with some screenshots.

Oh #!$& !
I just asked ocabot to merge https://github.com/OCA/pos/pull/682, before reading this :/

**Review by ivantodorovich — APPROVED** [review]
Technically looks good!

**Comment by legalsylvain** [comment]
Thanks @paradoxxxzero  for the screenshots.

> Oh #!$& !
> I just asked ocabot to merge #682, before reading this :/

Not a big deal, maybe we can add a small text in the ROADMAP.rst of the other module, to mention to port ``pos_receipt_hide_price`` in next version and not the other one ?

/ocabot merge patch

**Pull request merged**